esphome/components/bl0940/number/calibration_number.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "esphome/components/number/number.h"
+#include "esphome/core/component.h"
+#include "esphome/core/preferences.h"
+
+namespace esphome {
+namespace bl0940 {
+
+class CalibrationNumber : public number::Number, public PollingComponent {
```

Comment:
Does not need to be polling
```suggestion
class CalibrationNumber : public number::Number, public Component {
```